extism_pdk.go

```diff
@@ -208,13 +212,13 @@ func SetVar(key string, value []byte) {
 	defer keyMem.Free()
 
 	valMem := AllocateBytes(value)
-	defer valMem.Free()
```

Comment:
We should add this back since https://github.com/extism/js-sdk/pull/71 addresses the root of the issue